docs/image-based-upgrade.md

```diff
@@ -211,6 +211,49 @@ status:
   - Prep
 ```
 
+IBU status `.histoy` can be used understand when stages were desired and time taken to complete successfully
+
+```yaml
+status:
+  history:
+    - completionTime: "2024-06-26T03:27:05Z"
+      phases:
+        - completionTime: "2024-06-26T03:26:40Z"
+          phase: Stateroot
+          startTime: "2024-06-26T03:25:10Z"
+        - completionTime: "2024-06-26T03:27:05Z"
+          phase: Precache
+          startTime: "2024-06-26T03:26:41Z"
+      stage: Prep
+      startTime: "2024-06-26T03:25:07Z"
+    - completionTime: "2024-06-26T03:37:22Z"
+      phases:
+        - completionTime: "2024-06-26T03:27:37Z"
+          phase: PrePivot
+          startTime: "2024-06-26T03:27:36Z"
+        - completionTime: "2024-06-26T03:37:22Z"
+          phase: PostPivot
+          startTime: "2024-06-26T03:37:17Z"
+      stage: Upgrade
+      startTime: "2024-06-26T03:27:36Z"
+    - completionTime: "2024-06-26T03:48:12Z"
+      stage: Rollback
+      startTime: "2024-06-26T03:40:42Z"
+```
+
+Things to note from about `.status.history`
+
+- History is reset when desired stage is`Idle`.
+- Each history entry on the list will appear chronologically and will have `startTime` to indicated when a stage was desired for the first time. Optionally `completionTime` will appear when a stage completes successfully.
```

Comment:
What about when the stage fails?

Reply:
So if things fail we will only have startTime (and just reset after moving to `Idle`). 

It's similar to Job's status where [completionTime](https://github.com/kubernetes/api/blob/master/batch/v1/types.go#L526) only appears when it completes successfully.

Reply:
Yeah! Basically the way i was reading it...."stop the timer" with stopABC to complete. Do you want me to rename?

Reply:
don't bother if no other changes required